tests/test_polyhedron.py

```diff
@@ -739,12 +739,19 @@ def test_get_set_minimal_centered_bounding_sphere_radius(poly):
 )
 def test_is_inside(cube):
     assert cube.is_inside(cube.center)
+    import sys
 
-    @given(floats(0, 1), floats(0, 1), floats(0, 1))
-    def testfun(x, y, z):
+    limit = sys.float_info.min
```

Comment:
Let's use numpy instead and remove the `sys` import. This is also clearer (smallest normal is a more precise term than "min"). On my system these appear to give the same value.
```suggestion
    limit = np.finfo(np.float64).smallest_normal
```

Reply:
I will also update point in polygon test to use the same logic.